sdk/ai/ai-projects/src/agents/files.ts

```diff
@@ -31,6 +31,23 @@ export async function uploadFile(
   return result.body; 
 }
 
+/** Uploads a file for use by other operations. */
+export async function uploadFileAndPoll(
+context: Client, options: UploadFileParameters, sleepInterval: number,
```

Comment:
Since sleep number is an optional param for the user you can set the default here with `sleepInterval: number=1,`

Reply:
I updated this to have a default value of 1000 ms